test/xhtml/core/utils/get-selector.js

```diff
@@ -0,0 +1,24 @@
+
+describe('axe.utils.getSelector', function () {
```

Comment:
It's a little strange to have a test for `get-selector` duplicated in another directory. Can you make it work in the existing core unit tests by creating a document like in `test/core/utils/is-xhtml.js`?

Reply:
see explanation in the general comment below 👇

Reply:
This makes sense, although I'd suggest we move it to `tests/integrations` instead.